components/ui/AppCard.vue

```diff
@@ -14,9 +14,14 @@
     />
     <div class="app-card__content">
       <header class="app-card__header">
-        <h4 class="app-card__title">
-          {{ title }}
-        </h4>
+        <div>
+          <h4 class="app-card__title">
+            {{ title }}
+          </h4>
+          <h4 v-if="subtitle" class="app-card__subtitle">
```

Comment:
The `h` HTML tags [corresponds to the levels of nested sections](https://html.spec.whatwg.org/multipage/sections.html#the-h1,-h2,-h3,-h4,-h5,-and-h6-elements:~:text=corresponds%20to%20the%20levels%20of%20nested%20sections), so I think that if we have a title = `h4`, the subtitle should be `h5`

Reply:
Thanks! Done